codalab/worker/worker_run_state.py

```diff
@@ -265,6 +307,7 @@ def mount_dependency(dependency, shared_file_system):
                 status_message += "(and downloading %d other dependencies and docker images)" % len(
                     status_messages
                 )
+            logger.warning(f'{run_state.bundle.uuid} is not ready yet since {status_message}')
```

Comment:
Why is this a warning? Also, is this necessary to log? We don't want to overcrowd the logs.